recirq/quantum_chess/quantum_board.py

```diff
@@ -644,6 +652,8 @@ def do_move(self, m: move.Move) -> int:
                 self.circuit.append(
                     qm.split_slide(squbit, tqubit, tqubit2, path1, path2,
                                    ancilla))
+                self._clear_path_ancilla(path_qubits, path1)
```

Comment:
This is not part of the PR but we should probably keep underscores consistent between create_path_ancilla and _clear_path_aniclla.  Probably correct is to add the underscore in create_path_ancilla.

Reply:
Renamed to `_create_path_ancilla`.